modules/apps/object/object-test/src/testFunctional/paths/ObjectField.path

```diff
@@ -137,6 +137,16 @@
 		<td>//button[@class='dropdown-item']//div[(text()="${key_fieldType}")]</td>
 		<td></td>
 	</tr>
+	<tr>
+		<td>TIME_STORAGE_OPTION</td>
+		<td>//button[contains(@class,'dropdown-item')]//div[(text()="${key_fieldTimeStorage}")]</td>
+		<td></td>
+	</tr>
```

Comment:
Out of curiosity, can we use an existing path for this dropdown menu option? We should a generic path for this